### ISSUE TITLE
[Bug]: Toast notifications should have an "x" button to close them

### ISSUE THREAD

**Issue by alexanderdd** [nextcloud/server#46616]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Notifications don't have a close button, so parts of the interface are not usable until the notification is gone (2sec?). Sometimes there are a lot of notifications or error messages, so a lot of the interface is covered/unusable.

![grafik](https://github.com/user-attachments/assets/528dbbd7-872a-44e1-8336-a60f3f5e9568)

### Steps to reproduce

1. do an action that creates a notification, for example delete Deck cards
2. notification is shown, no "x" button to close it
3. you can't use the interface behind the notification until the notification automatically disappears

### Expected behavior

there is an "x" button to close the notification

### Installation method

None

### Nextcloud Server version

29

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Additional info

@nextcloud/designers

**Comment by jancborchardt** [comment]
Agreed – this seems to be for the Vue components, although I can not find a corresponding component @susnux @skjnldsv @ShGKme

**Comment by ShGKme** [comment]
> Agreed – this seems to be for the Vue components, although I can not find a corresponding component 

No, this is `@nextcloud/dialogs` which uses `toastify-js` library.

**Comment by joshtrichards** [comment]
Duplicate of nextcloud-libraries/nextcloud-dialogs#352
